cinema/models.py

```diff
@@ -134,9 +137,8 @@ def save(
         )
```

Comment:
In your `save` method, you're calling `self.full_clean()`. This could potentially lead to performance issues because it forces Django to run all validations on the model, even those that are not necessary for the save operation. Consider removing this line if it's not necessary.